src/Stripe.net/Services/Accounts/AccountCapabilitiesAuBecsDebitPaymentsOptions.cs

```diff
@@ -0,0 +1,16 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class AccountCapabilitiesAuBecsDebitPaymentsOptions : INestedOptions
+    {
+        /// <summary>
+        /// Passing true requests the capability for the account, if it is not
```

Comment:
This should be `<c>true</c>` instead but that will mess your indentation for all comments 😓